containers/Reports/ReportCreateMessage.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Alert, View, Platform, KeyboardAvoidingView } from 'react-native';
+import { Button } from '@rneui/base';
+import { Formik } from 'formik';
+import * as yup from 'yup';
+import IconSLI from 'react-native-vector-icons/SimpleLineIcons';
+import { ButtonSubmit } from '../../components/ButtonSubmit';
+import { TextSourceSans } from '../../components/TextSourceSans';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import API from '../../BaseApi';
+
+import { styles } from '../Ideas/Idea.styles';
+import { TextInputFullFormField } from '../../components/formFields';
+
+export const ReportCreateMessage = props => {
+  const {content_type, object_pk} = props.route.params;
+
+  const arrowLeftIcon = (
+    <IconSLI name='arrow-left' size={22} />
+  );
+  const reportMessageValidationSchema = yup.object().shape({
+    message: yup
+      .string()
+      .max(1024, 'Message must be no longer then 1024 characters')
+  });
+
+  const handleSubmit = (values) => {
+    let description;
```

Comment:
this can be a const as it doesn't change if you do `const description = values.message` instead of two lines